app/src/NativeBase/Components/Project/ProjectRoleTitle.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file shows the project role title in a coloured box for use in projects
+ */
+
+import React from 'react'
+import { Box } from 'native-base'
+
+type ProjectRoleProps = {
+  role: string
+}
+
+/**
+ * A functional component to display the project role title in a coloured tag
+ * @param {ProjectRoleProps} props - the props for this component
+ * @param {string} props.role - the role title
+ * @returns {JSX.Element|null} Renders the project role in a coloured tag, or null if no role
+ */
+const ProjectRoleTitle = ({ role }: ProjectRoleProps) => {
+  const colours = ['blue.20', 'purple.40', 'primary.60', 'primary.40']
+  const randomColour = colours[Math.floor(Math.random() * colours.length)]
```

Comment:
It just looks random in the designs.. We could ask and change later if so desired?

Reply:
It would be better to do it now.  But if you don't have time to and you can otherwise get this PR done before you go away, you could add a separate Jira ticket to tackle this afterwards.